lib/path_manager.c

```diff
@@ -74,21 +74,22 @@ bool mptcpd_pm_ready(struct mptcpd_pm const *pm)
         return pm->family != NULL;
 }
 
-int mptcpd_pm_add_addr(struct mptcpd_pm *pm,
-                       struct sockaddr const *addr,
-                       mptcpd_aid_t address_id,
-                       uint32_t flags,
-                       int index,
-                       mptcpd_token_t token)
+// -------------------------------------------------------------------
+
+int mptcpd_kpm_add_addr(struct mptcpd_pm *pm,
+                        struct sockaddr const *addr,
+                        mptcpd_aid_t address_id,
+                        uint32_t flags,
+                        int index)
 {
         if (pm == NULL || addr == NULL || address_id == 0)
                 return EINVAL;
 
         if (!is_pm_ready(pm, __func__))
                 return EAGAIN;
 
-        struct mptcpd_pm_cmd_ops const *const ops =
-                pm->netlink_pm->cmd_ops;
+        struct mptcpd_kpm_cmd_ops const *const ops =
+                pm->netlink_pm->kcmd_ops;
 
         if (ops->add_addr == NULL)
```

Comment:
Similar functions have added an `ops == NULL` check here

Reply:
Good catch.  I'll add the missing `NULL` pointer check.